<launch>
  <arg name="cam_name" />
  <arg name="calib_file" default="$(find svo_ros)/calib/$(arg cam_name).yaml"/>

  <node pkg="nodelet" type="nodelet" name="vo_nodelet_manager"  args="manager" output="screen"/>

  <!-- SVO nodelet -->
  <node pkg="nodelet" type="nodelet" name="svo" clear_params="true" args="load svo_ros/SvoNodelet vo_nodelet_manager" output="screen">

    <!-- Camera topic to subscribe to -->
    <param name="cam0_topic" value="camera/image_raw" type="str" />

    <!-- Camera calibration file -->
    <param name="calib_file" value="$(arg calib_file)" />

    <!--Parameters-->
    <!-- <rosparam file="$(find svo_ros)/param/pinhole.yaml" /> -->
    <rosparam file="$(find svo_ros)/param/fast_pinhole.yaml" />
    <param name="publish_img_pyr_level" value="2" />
    <param name="publish_every_nth_img" value="10" />

  </node>
</launch>
